client/my-sites/plugins/plugins-list/index.jsx

```diff
@@ -455,5 +456,9 @@ export default connect(
 		selectedSite: getSelectedSite( state ),
 		selectedSiteSlug: getSelectedSiteSlug( state ),
 	} ),
-	{ recordGoogleEvent }
+	{ recordGoogleEvent },
+	null,
+	{
+		withRef: true
```

Comment:
For testing purposes we don't need this. We can instead export the bare class, along with the connected component:
```
export PluginsList;
```